Kudu.Core/Infrastructure/ZipArchiveExtensions.cs

```diff
@@ -171,5 +178,11 @@ private static string EnsureTrailingSlash(string input)
         {
             return input.EndsWith("/", StringComparison.Ordinal) ? input : input + "/";
         }
+
+        public static bool IsChildOf(this string path, string parentPath)
```

Comment:
private?

Reply:
This doesn't seem to be the right extension for a string, a function or inline code should be fine.